conversation/csharp/sdk/conversation/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Dapr.AI.Conversation;
+using Dapr.AI.Conversation.Extensions;
+
+class Program
+{
+  private const string ConversationComponentName = "echo";
+
+  static async Task Main(string[] args)
+  {
+    const string prompt = "What is dapr?";
+
+    var builder = WebApplication.CreateBuilder(args);

```

Comment:
@WhitWaldo can you review the Conversation Client instantiation here? Should I do it a different way?

Also, is it weird putting the builder/app in a Class? Can remove that...

Reply:
This is precisely right. As this is configured to use a "Microsoft.NET.Sdk.Web" project, this is the appropriate way to set up the "app builder" itself. Dapr then gets registered on line 28, DI finalized on line 29 and the Dapr client instantiated from that DI pipeline on 32.

Technically this could be shortened a little bit as single-file C# apps don't as much of the window dressing any longer, but this is fine - if you want to shorten it, remove lines 19, 20 and 54 (`class Program { }`)